test/unit/TransferTransaction.js

```diff
@@ -59,4 +62,90 @@ describe("TransferTransaction", function () {
             transferTransactionFromBytes.nftTransfers.values()
         );
     });
+
+    it("should use nftid case for addNftTransfer", function () {
+        let transferTransaction = new TransferTransaction();
+
+        let tokenId = new TokenId(1, 2, 3);
+        let serial = Long.fromString("1234567890");
+
+        let nftId = new NftId(tokenId, serial);
+        let sender = AccountId.fromString("1.1.1");
+        let recipient = AccountId.fromString("2.2.2");
+
+        let check = {
+            serial:
+                typeof serial === "number" ? Long.fromNumber(serial) : serial,
```

Comment:
Not sure why you have these `typeof` conditions here when you know the type already.